docs/source/admin/traffic_router.rst

```diff
@@ -447,14 +447,14 @@ The following is an example of the request flow when a client requests the routi
 
 Troubleshooting and log files
 =============================
-Traffic Router log files can be found under :file:`/opt/traffic_router/var/log` and :file:`/opt/tomcat/logs`. Initialization and shutdown logs are in :file:`/opt/tomcat/logs/catalina{date}.out`. Application related logging is in :file:`/opt/traffic_router/var/log/traffic_router.log`, while access logs are written to :file:`/opt/traffic_router/var/log/access.log`.
+Traffic Router log files can be found under :file:`/var/log/traffic_router` and :file:`/var/log/tomcat`. Initialization and shutdown logs are in :file:`/var/log/tomcat/catalina{date}.out`. Application related logging is in :file:`/var/log/traffic_router/traffic_router.log`, while access logs are written to :file:`/var/log/traffic_routeraccess.log`.
```

Comment:
/var/log/traffic_routeraccess.log -> /var/log/traffic_router/access.log

Reply:
Good catch, thanks! Fixed in 72834a25bc.